app/models/inventory_list_item.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class InventoryListItem < ApplicationRecord
```

Comment:
Basically all the contents of this class will eventually be extracted to the `Listable` concern since it's all duplicated between this and `ShoppingListItem`.